crates/api_models/src/admin.rs

```diff
@@ -186,6 +193,9 @@ pub struct MerchantAccountResponse {
     /// An identifier for the vault used to store payment method information.
     #[schema(example = "locker_abc123")]
     pub locker_id: Option<String>,
+    ///Default business details for connector routing
+    #[schema(value_type = Option<PrimaryBusinessDetails>)]
+    pub primary_business_details: serde_json::Value,
```

Comment:
```suggestion
    pub primary_business_details: pii::SecretSerdeValue,
```

Reply:
This is in response, do we need to have secret value in all the other places?

Reply:
yes